### ISSUE TITLE
代码运用到自己的模型

### ISSUE THREAD

**Issue by Tangzhaotz** [hyz-xmaster/swa_object_detection#2]
怎么把这种方法运用到自己的训练中？仅仅多训练12个epochs吗？

**Comment by hyz-xmaster** [comment]
Yes, but you need use cyclical learning rates for the extra training and then average these 12 checkpoints.

**Comment by Tangzhaotz** [comment]
where the code for this part?

**Comment by hyz-xmaster** [comment]
Please see the [Getting started](https://github.com/hyz-xmaster/swa_object_detection#getting-started).

**Comment by Tangzhaotz** [comment]
ok，thank you

**Comment by Mobu59** [comment]
> Yes, but you need use cyclical learning rates for the extra training and then average these 12 checkpoints.

hello ,I use the get_swa_model.py and CosineAnnealingLR（initial learning rate 1e-3,eta_min=1e-4）to train my best model for 12 extra epochs,and then get avg_model.But when I use the avg_model to test, map decreased...
